kyo-core/shared/src/main/scala/kyo/Duration.scala

```diff
@@ -0,0 +1,136 @@
+package kyo
+
+import java.time.Duration as JavaDuration
+import java.time.temporal.ChronoUnit.*
+import kyo.Duration.Units
+import kyo.Duration.Units.*
+import kyo.duration.*
+import scala.concurrent.duration.Duration as ScalaDuration
+
+type Duration = duration.Duration
+given CanEqual[Duration, Duration] = canEqual
```

Comment:
Can you move this to the companion object?

Reply:
Will do

Reply:
I don't think it's possible. The type is necessary to be in the top level namespace for the extension methods.